src/scenes/NftDetailPage/NavigationHandle.tsx

```diff
@@ -28,9 +28,12 @@ function NavigationHandle({ direction, nftId }: Props) {
 
   const hoverText = useMemo(() => HOVER_TEXT.get(direction) ?? '', [direction]);
 
+  const { push } = useRouter();
   const handleOnClick = useCallback(() => {
-    void navigate(nftId, { state: { collection: [] } });
-  }, [nftId]);
+    // TODO(Terence): Figure out how to get this state across since Next doesn't support navigation state
+    // void push(nftId, { state: { collection: [] } });
+    void push({ pathname: nftId });
```

Comment:
@Robinnnnn Can we pair up on this? It looks like this component is commented out and not being used so maybe we can just delete it?

Reply:
this is a fine for now, don't worry about it, we do eventually wanna use this component